src/neo/SmartContract/Native/ContractManagement.cs

```diff
@@ -172,13 +180,7 @@ private ContractState Deploy(ApplicationEngine engine, byte[] nefFile, byte[] ma
 
             engine.Snapshot.Add(key, new StorageItem(contract));
 
-            // Execute _deploy
-
-            ContractMethodDescriptor md = contract.Manifest.Abi.GetMethod("_deploy", 2);
-            if (md != null)
-                engine.CallFromNativeContract(Hash, hash, md.Name, data, false);
-
-            engine.SendNotification(Hash, "Deploy", new VM.Types.Array { contract.Hash.ToArray() });
+            OnDeploy(engine, contract, data, false);
```

Comment:
require await?

Reply:
No. It's a normal method.

Reply:
"OnDeploy" is asynchronous method now, if not await, we 
 will lose control of the method's definite execution, which may cause unpredictable runtime errors.